app/main.py

```diff
@@ -1 +1,46 @@
-#  write your code here
+class Animal:
+    def __init__(self, name: str,
```

Comment:
fix indentation here, one arg per line

Reply:
still not fixed. Move name to the new line